### DISCUSSION TITLE
애드가드 필터로컬호스트 호환성 질문

### DISCUSSION THREAD

**Discussion by Keum-KangSeon-LostArkDirectorFan** [List-KR/community#25] [Q&A]
list kr질문은 아니지만 타사이트에 댓글다신걸 보니 애드가드를 잘아시는것 같아서 질문드립니다.
고급설정의 필터로컬호스트를 켜면 아프리카TV 사이트와 고화질이상의 재생을 위한 AfreecaTvPackage.exe(백그라운드)와의 소통? 연결?이 방해가되서 안켜진걸로 판단되는지 고화질재생이 안됩니다
애드가드에 문의해서 답변받은것들로 해결이 안되서요 한국인이시니 해결방법을 아실지도 모를까싶어서요...

**Comment by piquark6046** [comment]
AdGuard for Windows 7.13을 사용하시고 계신가요?
만약 그렇다면, 아무 아프리카TV 방송에서 플레이어의 `진단하기` 버튼으로 확인해보시겠어요?

또한, AhnLab V3 같은 안티 바이러스, 금융 보안 프로그램, 회사 보안 프로그램나 방화벽이 설치되어 있다면 알려주세요.

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
앱체크와 이셋백신 사용중입니다

**Reply by piquark6046** [reply]
ESET 안티 바이러스와 AdGuard 간의 충돌이 있는 걸로 알고 있습니다.
ESET 안티 바리어스의 방화벽 기능와 네트워크 기반 감시 기능을 끄고 재부팅 후에 다시 해보세요.

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
이셋백신과의 호환성문제는 아니네요...
애드가드의 각항목별로 꺼봐도 마찬가지인걸 보니 무조건 필터로컬호스트가 문제네요

**Reply by piquark6046** [reply]
제거까지 하셨나요?
Microsoft Defender, 아프리카TV 그리드 컴퓨팅 프로그램와 AdGuard만 설치되어 있을 때 아프리카TV 고화질이 잘 재생됩니다.
전 AppCheck 일반 버전도 테스트해보겠습니다.

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
필터로컬호스트키고 진단하니 고화질 재생문제:프로그램이 설치되있나요? 혹은 21201 포트를 확인하세요라네요

**Reply by piquark6046** [reply]
ESET 안티 바이러스 설치 없이는 여전히 잘 재생됩니다.

<details><summary>Screenshots</summary>

![Screenshot from 2023-05-21 02-42-00](https://github.com/List-KR/List-KR/assets/98787049/f050cd25-9086-4ce4-8502-9607f6a6e03d)
![Screenshot from 2023-05-21 02-45-56](https://github.com/List-KR/List-KR/assets/98787049/a79e9ef1-905f-49ad-b0ad-eb788528d6d3)


</details>

**Reply by piquark6046** [reply]
한번 ESET 제거 후 재부팅까지 해보세요.

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
해봐도 똑같습니다... 무언가 설정이 이상한걸까요?

**Reply by piquark6046** [reply]
AdGuard for Windows 설정이 어떻게 되어있나요?
Windows 제어판에 있는 프로그램 목록이 어떻게 되어 있나요?

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
<details><summary>Program List</summary>

![20230521_030744](https://github.com/List-KR/List-KR/assets/111874247/56a91812-5e44-4e35-b4f8-a145fb04ea41)
![20230521_030753](https://github.com/List-KR/List-KR/assets/111874247/68de7fdf-9db8-4e95-be6a-9f43c24f6eb6)
![20230521_030800](https://github.com/List-KR/List-KR/assets/111874247/56470859-6725-4d47-9717-328a9aef1550)
![20230521_030807](https://github.com/List-KR/List-KR/assets/111874247/e2a164f0-afc7-45a8-bd45-df6ef7643fad)
![20230521_030814](https://github.com/List-KR/List-KR/assets/111874247/7ff9f44e-364d-4bca-8ed5-35995ea508b6)


</details>

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
설정파일입니다. 확장자를 adg로 바꾸면됩니다
![20230521_030844_adg_settings_7 13 4266 0](https://github.com/List-KR/List-KR/assets/111874247/2a04fd54-6354-4191-b824-e7bc0ae214bc)

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
자야될 것 같습니다 할머니께서 얼른자라고 하시네요 ㅠㅠ 귀찮게 만들어서 죄송합니다

**Reply by piquark6046** [reply]
직접 선택하신 AdGuard의 일부 고급 설정이 문제를 발생시킴을 확인했습니다.
파악되는 동안 AdGuard를 초기화하시면 즉시 해결하실 수 있습니다.

**Reply by piquark6046** [reply]
`일반 HTTP 요청에 추가 공백 추가`를 비활성화해보세요.

**Reply by Keum-KangSeon-LostArkDirectorFan** [reply]
정말감사합니다 분명히 저번에 리디렉션을 꺼봤었는데도 해결이안됬었는데요 선생님이 말씀주신 옵션과 리디렉션을 끄니 해결됬습니다!! 몇달만에 해결했네요 ㅠㅠ 정말감사합니다!!
말씀주신옵션같은걸 킨적이없는데요... 옛날버전에서 패치하면서 켜진상태로 넘어와졌나 봅니다
